extensions/amp-carousel/0.1/base-carousel.js

```diff
@@ -142,7 +142,7 @@ export class BaseCarousel extends AMP.BaseElement {
   }
 
   /** @override */
-  documentInactiveCallback() {
+  unlayoutCallback() {
```

Comment:
Not sure if carousel supports auto-playing the slides or the items inside the carousel, if it does, one thing pausing that would be good to do in the `pauseCallback`.